snippets/route-navigate-with-matrix-params.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Route navigate with matrix params
```

Comment:
I suggest using word "optional" for these parameters

title might be "Optional parameters in the middle"